### PR TITLE
Changing services for aws deployment

### PR DESCRIPTION
**Pull request by aberke220**
- Add prod profiles to data acess services with db access params as env variables
- Integrate credit-application-service to graphql api of data-service
- Update tests and mock data format to conform graphql api results
- Create test and prod profiles for credit-application-service
- Related to [AB#2284](https://deloittedigitalat.visualstudio.com/38398c5c-11b2-4e64-98b5-5af5e65d6e82/_workitems/edit/2284)

### CODE REVIEW

File: domain-service/credit-application/src/main/java/at/deloittedigital/core_banking/credit/datasource/RemoteCreditDataService.java [hunk 1]
```diff
@@ -7,17 +7,27 @@
...
 import org.springframework.web.client.RestTemplate;
-import org.springframework.web.util.UriComponentsBuilder;
 
 @Service
 @Log4j2
 @ConditionalOnProperty(name = "core-banking.credit.datasource.mode", havingValue = "remote")
 public class RemoteCreditDataService implements CreditDataService {
 
+    // todo hardcoded graphql request for testing
+    private static final String TEST_QUERY =
```
Comment (theqajedi) [inline domain-service/credit-application/src/main/java/at/deloittedigital/core_banking/credit/datasource/RemoteCreditDataService.java:26]: sorry, did you mean creating DTO's for the queries in the todo?

Comment (aberke220) [reply domain-service/credit-application/src/main/java/at/deloittedigital/core_banking/credit/datasource/RemoteCreditDataService.java:26]: Yes, something that makes sense as a graphql API client. In this PR my goal was to integrate domain svc <-> data svc with some sample query that gets client info, and for that I used this hardcoded string query.
In the future I'd like to create something more OO for it that can be used to handle the various queries we need for the credit checks. I added AB#2337 for it.

**Review by theqajedi — APPROVED** [review]

**Comment by theqajedi** [comment]
any idea why the CI was not triggered for this PR?

**Comment by theqajedi** [comment]
so CI was triggered, but a bunch of runs is queued for whatever reason, not able to cancel them as well.
https://github.com/DeloitteDigitalAT/core-banking/actions/runs/609992115

**Pull request merged**